athena/tests/modules/programming/module_programming_llm/mock/utils/mock_llm.py

```
@@ -0,0 +1,36 @@
+        # Map keywords → raw JSON strings
+        self.responses: Dict[str, str] = responses or {}
+
+    async def predict(self, messages: List[Dict[str, str]], **kwargs) -> str:
+        system_message = next((msg["content"] for msg in messages if msg["role"] == "system"), "")
+        
+        if system_message in self.responses:
+            return self.responses[system_message]  
+        elif "Create graded feedback" in system_message:
+            return """{"type": "Assessment", "title": "Logic Error", "description": "Doc important", "file_path": "test.py", "line_start": 1, "line_end": 1, "credits": 0.5}"""
+        elif "Create non graded improvement suggestions" in system_message:
+            return """{"type": "Improvement", "title": "Logic Error", "description": "Doc important", "file_path": "test.py", "line_start": 1, "line_end": 1}"""
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)